The sprouts is failing to build the project and failing with following exception. It seems the machine where sprouts is running has java 11 and the bundle does not build with that.

```
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Skipping Apache Sling Distribution Core
[INFO] This project has been banned from the build due to previous failures.
[INFO] ------------------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.609 s
[INFO] Finished at: 2021-09-13T10:49:17Z
[INFO] ------------------------------------------------------------------------
[WARNING] The requested profile "ci" could not be activated because it does not exist.
[INFO] [jenkins-event-spy] Generated /home/jenkins/workspace/he-sling-distribution-core_PR-55@tmp/withMaven442fa326/maven-spy-20210913-104911-2979981949278491312987.log
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (set-bundle-required-execution-environment) on project org.apache.sling.distribution.core: An Ant BuildException has occured: Unable to create javax script engine for javascript
[ERROR] around Ant part ...<script language="javascript">var System = java.lang.System;... @ 4:33 in /home/jenkins/workspace/he-sling-distribution-core_PR-55/target/antrun/build-main.xml
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/src/main/java/org/apache/sling/distribution/agent/impl/SimpleDistributionAgentQueueProcessor.java
+++ b/src/main/java/org/apache/sling/distribution/agent/impl/SimpleDistributionAgentQueueProcessor.java
@@ -92,7 +92,7 @@ class SimpleDistributionAgentQueueProcessor implements DistributionQueueProcesso
 
             final long endTime = System.currentTimeMillis();
 
-            distributionLog.info("[{}] ITEM-PROCESSED item={}, status={}, processingTime={}ms", queueName, queueItem, success, endTime - startTime);
+            distributionLog.debug("[{}] ITEM-PROCESSED item={}, status={}, processingTime={}ms", queueName, queueItem, success, endTime - startTime);
 
             return success;
 
@@ -147,8 +147,8 @@ class SimpleDistributionAgentQueueProcessor implements DistributionQueueProcesso
                     removeItemFromQueue = true;
                     final long endTime = System.currentTimeMillis();
 
-                    distributionLog.info("[{}] PACKAGE-DELIVERED {}: {} paths={}, importTime={}ms, execTime={}ms, size={}B", queueName, requestId,
-                            requestType, paths,
+                    distributionLog.info("[{}] PACKAGE-DELIVERED {}: {} item={}, paths={}, importTime={}ms, execTime={}ms, size={}B", queueName, requestId,
+                            requestType, queueEntry.getItem().getPackageId(), paths,
                             endTime - startTime, endTime - globalStartTime,
                             packageSize);
                 } catch (RecoverableDistributionException e) {

--- a/src/main/java/org/apache/sling/distribution/queue/DistributionQueueItem.java
+++ b/src/main/java/org/apache/sling/distribution/queue/DistributionQueueItem.java
@@ -37,7 +37,6 @@ public final class DistributionQueueItem extends ValueMapDecorator implements Va
 
     private final String packageId;
     private final long size;
-    private final Map<String, Object> base;
 
     public DistributionQueueItem(@NotNull String packageId, Map<String, Object> base) {
         this(packageId, -1, base);
@@ -47,7 +46,6 @@ public final class DistributionQueueItem extends ValueMapDecorator implements Va
         super(base);
         this.packageId = packageId;
         this.size = size;
-        this.base = base;
     }
 
     @NotNull
@@ -67,25 +65,25 @@ public final class DistributionQueueItem extends ValueMapDecorator implements Va
     public String toString() {
         return "DistributionQueueItem{" +
                 "id='" + packageId + '\'' +
-                ", info={" + queueInfo(base) + '}' +
+                ", info={" + queueInfo() + '}' +
                 '}';
     }
 
     /*
      * convert the map of object values into string form
      */
-    private String queueInfo(Map<String, Object> base) {
-        String queueItem = "";
-        for(String key : base.keySet()) {
-            Object value = base.get(key);
+    private String queueInfo() {
+        String queueItemStr = "";
+        for(String key : super.keySet()) {
+            Object value = super.get(key);
             String valueString = "";
-            if (value instanceof String[]) {
-                valueString = key + "=" + Arrays.toString((String[])value);
+            if (value.getClass().isArray()) {
+                valueString = key + "=" + Arrays.toString((Object[])value);
             } else {
                 valueString = key + "=" + value.toString();
             }
-            queueItem = String.join(",", queueItem, valueString);
+            queueItemStr = String.join(",", queueItemStr, valueString);
         }
-        return queueItem.isEmpty() ? queueItem : queueItem.substring(1);
+        return queueItemStr.isEmpty() ? queueItemStr : queueItemStr.substring(1);
     }
 }

--- a/src/main/java/org/apache/sling/distribution/queue/DistributionQueueItem.java
+++ b/src/main/java/org/apache/sling/distribution/queue/DistributionQueueItem.java
@@ -65,14 +65,14 @@ public final class DistributionQueueItem extends ValueMapDecorator implements Va
     public String toString() {
         return "DistributionQueueItem{" +
                 "id='" + packageId + '\'' +
-                ", info={" + queueInfo() + '}' +
+                ", info={" + getQueueInfo() + '}' +
                 '}';
     }
 
     /*
      * convert the map of object values into string form
      */
-    private String queueInfo() {
+    private String getQueueInfo() {
         String queueItemStr = "";
         for(String key : super.keySet()) {
             Object value = super.get(key);

